guides/using-helpers.md

```diff
@@ -4,4 +4,166 @@ layout: guides
 permalink: /guides/using-helpers/
 ---
 
-#Using Helpers
+# Using Helpers
+
+Helpers are extensions added to Dust to increase its functionality. They make it possible for anyone to extend the way Dust works. They are commonly used to add functionality related to things like logic, formatting, and internationalization. Helpers are denoted by the `@` symbol. Let's take a look at some commonly used helpers.
+
+## Installing Helpers
+
+Helpers are bundled separately as the `dustjs-helpers` library. After the dustjs-linkedin library is loaded, follow the same instructions in the [Getting Started guide](/guides/getting-started/) to install `dustjs-helpers`. If you are cloning or downloading from GitHub, you can find the helpers in the [dustjs-helpers repository](https://github.com/linkedin/dustjs-helpers).
```

Comment:
I'll add the note to the Getting Started guide, since you can also use it for the main library. What's the difference between dust-core and dust-full?